templates/essentials.yaml

```diff
@@ -340,6 +340,11 @@ Resources:
             Effect: "Allow"
             Principal:
               AWS:
+                - !Join
```

Comment:
why not just give admin role access?  https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/bootstrap.yaml#L160

if you still prefer root it's better to use a !Sub than a !Join like so..
```
  - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
```

Reply:
Does the admin role exist when this template is built?

Reply:
yes, bootstrap.yaml is the first template deployed to each account.

Reply:
but the admin role is output from accounts.yaml and so cannot be referenced by bootstrap.yaml... right?

Reply:
gah, you are correct!! there is no way to give admin role access to this key.  I guess this is ok 
 but root doesn't help because logging in with root requires MFA and i believe the MFA is kept under lock and key.   I see two options:
1. Create a new key in account.yaml and re-encrypt with secure string with the new key
2. All IT admins get the travis creds to update the secrets.

Reply:
I'll merge this, then. I think (1) is the best choice though I need to make sure that string isn't actually used in bootstrap.